.readthedocs.yaml

```diff
@@ -33,18 +33,8 @@ build:
     # Before building the docs, build JupyterGIS in its isolated environment,
     # then install the wheels into the docs environment.
     pre_build:
-      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn install"
-      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn dev"
-      - /bin/bash --login -c "micromamba run -n jupytergis-build jupyter labextension list 2>&1 | grep -ie 'jupytergis-core.*OK'"
-      - /bin/bash --login -c "micromamba run -n jupytergis-build jupyter labextension list 2>&1 | grep -ie 'jupytergis-lab.*OK'"
-      - /bin/bash --login -c "micromamba run -n jupytergis-build yarn build:packages"
-
-      - |-
-        /bin/bash --login -c "micromamba run -n jupytergis-docs \
-          python -m pip install \
-          $(ls ./python/jupytergis_core/dist/jupytergis*.whl) \
-          $(ls ./python/jupytergis_lab/dist/jupytergis*.whl) \
-          $(ls ./python/jupytergis_qgis/dist/jupytergis*.whl)"
```

Comment:
We should keep this? 

This was making a dev install of jupytergis for the docs and was working fine

Reply:
Yeah, I was trying to replicate my setup in https://github.com/davidbrochart/jupytergis-demo, which works. But let's try your suggestion.

Reply:
Feel free to push to this branch.

Reply:
I think it will be faster if you push changes directly instead of suggesting them and having me commit them.

Reply:
Dumb question: why don't we have the JupyterLite deployment on GitHub pages like in https://github.com/davidbrochart/jupytergis-demo, in the end we just have a link in our docs right? I don't really understand the need to build in RTD, if it's more complicated?

Reply:
After discussing with @martinRenou, the reason we build on RTD is that this allows to have a deployment per PR.

Reply:
> We should keep this?
> 
> This was making a dev install of jupytergis for the docs and was working fine

But:
- it builds `jupytergis_qgis`, which cannot work in WASM?
- it runs `python scripts/dev-install.py`, which in turn installs everything in `requirements-build.txt`, including `jupyter-collaboration`, which is going to clash with `my-jupyter-shared-drive`?

Reply:
Yeah, we should remove `jupytergis_qgis` and anything qgis-related from there.

You moved the jupyter-collaboration dependency right? I guess that script need to be updated a bit and simplified.

Also we probably need to pin micromamba to 2.0.5.

Reply:
I see jupyter-collaboration is part of [requirements-build.txt](https://github.com/geojupyter/jupytergis/blob/b4785da432584ccbed67ef6a5009f6c8baa7cd02/requirements-build.txt#L4), I suppose this is what you use in tests? That is why I didn't want to use `yarn dev` for the JupyterLite deployment, and in the end why I removed all the code above.

Reply:
Got it! Indeed using `yarn dev` may not be possible as-is.